new.js

```diff
@@ -0,0 +1 @@
+var password = "mypassword"
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/daltonmblack/node-proxy/pullRequest?prid=8027926)
```suggestion
var password = "mypassword";
```